test/cloud-metadata/_fixtures.js

```diff
@@ -64,128 +64,11 @@ module.exports = {
     },
   ],
   gcp: [
-    {
-      name: 'gcp does not crash on empty response',
-      response: {},
-    },
-    {
-      name: 'gcp unexpected string fixture',
-      response: {
-        instance: {
-          zone: 123456,
-          machineType: 123456,
-        },
-      },
-    },
     {
       name: 'default gcp fixture',
-      response: {
-        instance: {
-          attributes: {},
-          cpuPlatform: 'Intel Broadwell',
-          description: '',
-          disks: [
-            {
-              deviceName: 'username-temp-delete-me-cloud-metadata',
-              index: 0,
-              interface: 'SCSI',
-              mode: 'READ_WRITE',
-              type: 'PERSISTENT',
-            },
-          ],
-          guestAttributes: {},
-          hostname:
-            'username-temp-delete-me-cloud-metadata.c.elastic-apm.internal',
-          id: 7684572792595385000,
```

Comment:
Reviewer note: I've changed this test fixture to use an **unparsed** actual response on a test GCP VM. The problem with having an already parsed object, is that it accidentally included the BigInt loss of precision. That `7684572792595385000` is wrong. The actual instance.is was some `7684572792595385???` value with something other than zeros. It was just not noticed originally.